comfyui_to_python.py

```diff
@@ -10,9 +10,10 @@
 
 import black
 
-from utils import import_custom_nodes, add_comfyui_directory_to_sys_path, get_value_at_index
-
 sys.path.append('../')
+sys.path.append('.')
```

Comment:
I think this is not needed. The current working directory will already be added to path.

Reply:
yeah I thought so too but without it the generation did not work. Util's fle could not be imported.

Reply:
> PS J:\SD\ComfyUI\ComfyUI\ComfyUI-to-Python-Extension> ..\..\python_embeded\python.exe .\comfyui_to_python.py
Traceback (most recent call last):
  File "J:\SD\ComfyUI\ComfyUI\ComfyUI-to-Python-Extension\comfyui_to_python.py", line 16, in <module>
    from utils import import_custom_nodes, find_path, add_comfyui_directory_to_sys_path, get_value_at_index, add_extra_model_paths
ModuleNotFoundError: No module named 'utils'
PS J:\SD\ComfyUI\ComfyUI\ComfyUI-to-Python-Extension>